submodules/eks/iam.tf

```diff
@@ -237,6 +262,310 @@ data "aws_iam_policy_document" "custom_eks_node_policy" {
   ]
 }
 
+# https://raw.githubusercontent.com/kubernetes-sigs/aws-load-balancer-controller/v2.5.4/docs/install/iam_policy.json
```

Comment:
Is it easier to just keep a copy of this file instead of translating it to HCL?

Reply:
Having the json file will be easier I will check how to do it